README.md

```diff
@@ -44,15 +44,15 @@ The TestCafe version to install.
     args: "chrome tests"
 ```
 
-The **latest version** is installed by default.
+**Default value:** `latest` (the latest version)
```

Comment:
I'm sure 'latest' is self-explanatory and you do not need to add '(the latest version)'